tests/ui/expr/weird-exprs.rs

```diff
@@ -271,6 +272,24 @@ fn fish_fight() {
     tug_o_war(<T>::_____________::<T>);
 }
 
+// all 48 keywords in 300 characters
+#[allow(warnings)]
```

Comment:
Suggestion: I would not unconditionally allow all warnings. Cf. the explicit lints allowed on the top of the file, I would use explicit error annotations for this module.